### PR TITLE
[ASUYTP] Add quizz name to ytp api submission statics

### PR DESCRIPTION
**Pull request by arsentur**
**Description:** 
Added to ytp API submission statics quizz name filed

**Youtrack:** 
https://youtrack.raccoongang.com/issue/ASUYT-110

### CODE REVIEW

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

**Comment by raccoondeploy** [comment]
Can one of the admins verify this patch?

File: lms/djangoapps/ytp_api/backends/tracking.py [hunk 1]
```diff
@@ -62,6 +62,10 @@ def get_event_user(event):
     def get_event_course_id(event):
         return event.get('context', {}).get('course_id')
 
+    @staticmethod
+    def get_event_quizz_name(event):
```
Comment (idegtiarov) [inline lms/djangoapps/ytp_api/backends/tracking.py]: typo: s/get_event_quizz_name/get_event_quiz_name/

File: lms/djangoapps/ytp_api/backends/tracking.py [hunk 2]
```diff
@@ -75,9 +79,9 @@ def process(self, event):
         result['course_id'] = self.get_event_course_id(event)
         result['time'] = self.get_event_timestamp_as_string(event)
         result['username'] = self.get_event_user(event)
+        result['quizz_name'] = self.get_event_quizz_name(event)
```
Comment (idegtiarov) [inline lms/djangoapps/ytp_api/backends/tracking.py]: ditto

**Pull request merged**